metrics/web/test/base/presentation/graphs/colored_bar_test.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/graphs/colored_bar.dart';
+
+import '../../../test_utils/metrics_themed_testbed.dart';
+
+void main() {
+  group("ColoredBar", () {
+    testWidgets(
+      "applies the color of the bar",
+      (WidgetTester tester) async {
+        const color = Colors.red;
+
+        await tester.pumpWidget(const _ColoredBarTestbed(color: color));
+
+        final barContainer = tester.widget<Container>(find.descendant(
+          of: find.byType(ColoredBar),
+          matching: find.byType(Container),
+        ));
+
+        final barContainerDecoration = barContainer.decoration as BoxDecoration;
+
+        expect(barContainerDecoration.color, color);
+      },
+    );
+
+    testWidgets(
+      "applies the padding to the bar",
+      (WidgetTester tester) async {
+        const padding = EdgeInsets.all(32.0);
+
+        await tester.pumpWidget(const _ColoredBarTestbed(padding: padding));
+
+        final barPadding = tester.widget<Padding>(find.descendant(
+          of: find.byType(ColoredBar),
+          matching: find.byType(Padding),
+        ));
+
+        expect(barPadding.padding, padding);
+      },
+    );
+
+    testWidgets(
+      "applies the border decoration to the bar",
```

Comment:
would it be better if we specify precisely what decorations we can do? maybe splitting into 2 tests - one for border and another for the border-radius? Or we could pass decoration directly to the widget?